images/node/system-container/system-container-wrapper.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+source /run/$NAME-env
+
+exec /usr/local/bin/origin-node-run.sh
```

Comment:
if I don't use the `exec` the script will fork here and keep the shell process around